.github/workflows/index.yaml

```diff
@@ -0,0 +1,50 @@
+name: Update Package Metadata Index
+
+on:
+#    push:
+#        branches:
+#            - master
+#        paths:
+#            - 'meta/*'
+    schedule:
+        - cron: '*/30 * * * *'
+
+jobs:
+    index:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v1
+              with:
+                  php-version: 7.3
+                  extensions: json
+                  tools: prestissimo
+
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - uses: actions/cache@v1
+              with:
+                  path: indexer/vendor/
+                  key: ${{ runner.os }}-composer-vendor
+
+            - uses: actions/cache@v1
+              id: cache-files
+              with:
+                  path: indexer/cache/
+                  key: ${{ runner.os }}-cache-files
+
+            - name: Install the dependencies
+              run: cd indexer && composer install --no-interaction --no-suggest
+
+            # Update statistics twice a day (24 * 30 mins = 12h)
+            - name: with-stats
+              run: |
+                  echo "::set-env name=with_stats::$(expr ${{ github.run_number }} % 24)"
+
+            - name: Update Index
+              run: indexer/index ${{ env.with_stats == 0 && '--with-stats' || '' }}
```

Comment:
That's too often! We only update downloads and favs twice a month now. This will generate way too many index updates.